.github/workflows/deploy-lts-prow.yaml

```diff
@@ -0,0 +1,66 @@
+name: Deploy AKS LTS Prow
+on: [push, workflow_dispatch]
+jobs:
+  Deploy_AKS_LTS_Prow:
+    runs-on: ubuntu-latest
+    env:
+      GITHUB_APP_ID: ${{ vars.APP_ID }}
+      GITHUB_ORG: ${{ vars.ORG }}
+      GITHUB_REPO: ${{ vars.REPO }}
+      GITHUB_TOKEN: ${{ secrets.TOKEN }}
+      HMAC_TOKEN: ${{ secrets.HMAC_TOKEN }}
+      MINIO_CONSOLE_PORT: 8003
+      FAKE_MOUNT_SECRET: ${{ secrets.FAKE_MOUNT_SECRET }}
+    steps:
+      - name: Check out repo
+        uses: actions/checkout@v3
+
+      - name: Log into Azure
+        uses: azure/login@v1
+        with:
+          creds: ${{ secrets.AZURE_CREDENTIALS }}
+
+      - name: Deploy Prow cluster Bicep
+        id: bicep
+        uses: azure/arm-deploy@v1
+        with:
+          subscriptionId: ${{ vars.AZURE_SUBSCRIPTION }}
+          resourceGroupName: ${{ vars.AZURE_RG }}
+          template: ./config/prow/cluster/prow-cluster.bicep
+          failOnStdErr: false
+
+      - name: Fetch config
+        run: |
+          echo "PROW_HOST=${{ steps.bicep.outputs.prowHostName }}" >> "$GITHUB_ENV"
+          echo "AZURE_STORAGE_ACCOUNT_USER=${{ steps.bicep.outputs.storageAccountName }}" >> "$GITHUB_ENV"
+
+      - name: Fetch storage key
```

Comment:
I was hopping this can get from bicep output directly

Reply:
Not recommended: https://learn.microsoft.com/en-us/azure/azure-resource-manager/bicep/linter-rule-outputs-should-not-contain-secrets